Mesh_3/doc/Mesh_3/Concepts/MeshEdgeCriteria_3.h

```diff
@@ -41,23 +41,40 @@ Numerical type.
 */
 typedef unspecified_type FT;
 
+/*!
+Feature index type.
+*/
+typedef unspecified_type Index;
+
 /// @}
 
 /// \name Operations
 /// @{
 
 /*!
 
-Returns the value of the sizing field (i.e., the maximum edge length) at point `p`.
+Returns the value of the sizing field (i.e., the maximum edge length) at point `p`,
+lying on subcomplex of dimension `dim` and index `index`.
 */
-FT sizing_field(const Point_3& p);
+FT sizing_field(const Point_3& p, const int dim, const Index& index);
```

Comment:
> [!WARNING]  
> There is a breaking change, here.

Reply:
Actually this is more a "fixing change". The sizing field takes this 3 parameters in the code, but the concept did not reflect it

Reply:
I agree that is a fix. There might be a way to keep that backward-compatible with existing user code.

Reply:
Well Jane says the code expects 3 parameters so the concept was broken anyway

Reply:
> Well Jane says the code expects 3 parameters so the concept was broken anyway

Then that is a fix of the documentation, and not a breaking change. Thanks.